### ISSUE TITLE
According to the LICENSE you should at least reference my name

### ISSUE THREAD

**Issue by 0blu** [KeilaPallo/easyqueue#1]
Hey,
nice to see someone who is maintaining my project, but please mind my license and add a reference to my repository.

**Comment by wtfcxt** [comment]
Yep, I don't think he did, also noticed that it's your project xD
